### DISCUSSION TITLE
import the Login form

### DISCUSSION THREAD

**Discussion by LightMike** [Beakerboy/VBA-SQL-Library#39] [General]
Hello,
I have been trying to import the files into the project in many ways,
but every time I am importing the Login form it is imported as module
instead.
Import - file - "Login.frm"

When looking at the code of the module it looks like that:
I have no idea, what causes this behaviour. Thanks for feedback.

VERSION 5#
Begin {C62A69F0-16DC-11CE-9E98-00AA00574A4F} Login
   Caption = "Please Log In"
   ClientHeight = 1920
   ClientLeft = 120
   ClientTop = 465
   ClientWidth = 2295
   OleObjectBlob   =   "Login.frx":0000
   StartUpPosition = 1    'CenterOwner
End
Public Ready As Boolean

Private Sub Form_Load()
Me.Show
Ready = False
Call Wait
'Label1.Visible = True
End Sub

Public Function Wait()
Do While Ready = False
    DoEvents
Loop
End Function

Private Sub LoginButton_Click()
    Ready = True
    Me.Hide
End Sub

**Comment by Beakerboy** [comment]
Clone the repository locally.

open Ms Excel
Click Developer>Visual Basic
Right-Click the “VbaProject(Something.xlsm)” for your project

click “Import File” and browse to the repository.
Select “Login.frm” and click “Open”

**Reply by LightMike** [reply]
Hello Kevin,
that is what I did a dozen times in different variations.
Also drag and drop, but the frm always ends up as module.
Is it a Mac / Windows issue or a country thing (Germany on my side)?

Thanks for the support. 

That's how it looks..
![SQLlib](https://github.com/Beakerboy/VBA-SQL-Library/assets/71723869/bf42ab7d-aaa8-4c0a-b572-176b5bdff017)

Regards,
Michael

**Comment by Beakerboy** [comment]
Are you on Mac or Windows? I did it successfully on windows using a copy that was on my computer, not from the repository. I’ll try downloading the versions from GitHub to make sure they work.

**Reply by LightMike** [reply]
Hello, 
I am on Windows 10 Pro and Office 2016 / PC not Mac.

**Comment by Beakerboy** [comment]
I tried it and it did the same thing for me. If I had to guess, I’d say the frm file has unix line endings instead of windows.

**Comment by Beakerboy** [comment]
#40

**Reply by LightMike** [reply]
Hello Kevin,
great idea. Replaced LF by CR LF and it worked.

![Unix_LE_After](https://github.com/Beakerboy/VBA-SQL-Library/assets/71723869/3626675f-7809-43b4-acae-5b770351e737)
![Unix_LE_Before](https://github.com/Beakerboy/VBA-SQL-Library/assets/71723869/9106d4cc-5e4f-4f19-96d0-64b9962c8e2a)
![UserForm](https://github.com/Beakerboy/VBA-SQL-Library/assets/71723869/17b5cde3-88c3-4004-afb5-ce98b987a108)

Thanks a lot.
